src/gui/wizard/owncloudwizard.cpp

```diff
@@ -46,6 +47,9 @@ Q_LOGGING_CATEGORY(lcWizard, "nextcloud.gui.wizard", QtInfoMsg)
 OwncloudWizard::OwncloudWizard(QWidget *parent)
     : QWizard(parent)
     , _account(nullptr)
+#ifdef WITH_PROVIDERS
```

Comment:
No it's not necessary but it will then be around in builds that are build without provider support. I will remove it. Indeed it makes the code easier then:)